cpp/benchmarks/copying/shift.cu

```diff
@@ -64,15 +54,12 @@ static void BM_shift(benchmark::State& state)
 {
   cudf::size_type size   = state.range(0);
   cudf::size_type offset = size * (static_cast<double>(shift_factor) / 100.0);
-  auto idx_begin         = thrust::make_counting_iterator<cudf::size_type>(0);
-  auto idx_end           = thrust::make_counting_iterator<cudf::size_type>(size);
-
-  auto input = use_validity
-                 ? cudf::test::fixed_width_column_wrapper<int>(
-                     idx_begin,
-                     idx_end,
-                     thrust::make_transform_iterator(idx_begin, [](auto idx) { return true; }))
-                 : cudf::test::fixed_width_column_wrapper<int>(idx_begin, idx_end);
+
+  auto const input_table =
+    create_sequence_table({cudf::type_to_id<int>()},
+                          row_count{size},
+                          use_validity ? std::optional<float>{1.0} : std::nullopt);
```

Comment:
FWIW you shouldn't need to specify the type here (or in the many other places that you use this parameter) Not a big deal though.
```suggestion
                          use_validity ? {1.0} : std::nullopt);
```

Reply:
Already tried this. Clang shows error that initializer list cannot be used on the right side of operator.
I wish there is a better way too.

Reply:
Sad! But thanks.